### ISSUE TITLE
已放弃 (核心已转储)

### ISSUE THREAD

**Issue by zhangshenghang** [PaddlePaddle/Paddle#57078]
### 请提出你的问题 Please ask your question

版本：
![image](https://github.com/PaddlePaddle/Paddle/assets/29418975/1e3cc939-cd9b-4743-ab6b-461f07f441c4)

cuDNN Version: 8.5
cuda Version: 11.7
![image](https://github.com/PaddlePaddle/Paddle/assets/29418975/f3b4cea8-6c3c-40db-8b1b-85900afbfff7)

使用的PaddleNLP官方提供的样例数据跑的程序
问题一：
使用命令：`python finetune.py   --train_path ./data2/train.json   --dev_path ./data2/dev.json   --save_dir ./checkpoint   --learning_rate 1e-5   --batch_size 16   --max_seq_len 512   --num_epochs 3   --model uie-senta-base   --seed 1000   --logging_steps 10   --valid_steps 100   --device gpu` 

使用GPU训练是报错：提示`已放弃 (核心已转储)`但是没报任何异常

.../sentiment_analysis/unified_sentiment_extraction python finetune.py   --train_path ./data2/train.json   --dev_path ./data2/dev.json   --save_dir ./checkpoint   --learning_rate 1e-5   --batch_size 16   --max_seq_len 512   --num_epochs 3   --model uie-senta-base   --seed 1000   --logging_steps 10   --valid_steps 100   --device gpu
/root/anaconda3/envs/nlp/lib/python3.8/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-09-07 17:47:02,836] [    INFO] - We are using (<class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'>, False) to load 'uie-senta-base'.
[2023-09-07 17:47:02,836] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/ernie_3.0_base_zh_vocab.txt
[2023-09-07 17:47:02,863] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/uie-senta-base/tokenizer_config.json
[2023-09-07 17:47:02,863] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/uie-senta-base/special_tokens_map.json
[2023-09-07 17:47:02,864] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-07 17:47:02,864] [    INFO] - Loading weights file model_state.pdparams from cache at /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-07 17:47:03,514] [    INFO] - Loaded weights file from disk, setting weights to model.
W0907 17:47:03.520568 63625 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0907 17:47:03.520634 63625 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.0, Runtime API Version: 11.7
W0907 17:47:03.528393 63625 gpu_resources.cc:149] device: 0, cuDNN Version: 8.5.
已放弃 (核心已转储)

![image](https://github.com/PaddlePaddle/Paddle/assets/29418975/fa80c3c1-7064-46ce-8ca5-39c52aa1bd45)

GPU使用：
![image](https://github.com/PaddlePaddle/Paddle/assets/29418975/c817dbc7-0640-4691-9be5-acd4a333403c)

问题二：
使用命令`python -u -m paddle.distributed.launch --gpus "0" finetune.py   --train_path ./data/train.json   --dev_path ./data/dev.json   --save_dir ./checkpoint   --learning_rate 1e-5   --batch_size 16   --max_seq_len 512   --num_epochs 3   --model uie-senta-base   --seed 1000   --logging_steps 10   --valid_steps 100   --device gpu`运行

报错

LAUNCH INFO 2023-09-07 17:49:15,903 ------------------------- ERROR LOG DETAIL -------------------------
/root/anaconda3/envs/nlp/lib/python3.8/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-09-07 17:49:13,939] [    INFO] - We are using (<class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'>, False) to load 'uie-senta-base'.
[2023-09-07 17:49:13,939] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/ernie_3.0_base_zh_vocab.txt
[2023-09-07 17:49:13,968] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/uie-senta-base/tokenizer_config.json
[2023-09-07 17:49:13,969] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/uie-senta-base/special_tokens_map.json
[2023-09-07 17:49:13,970] [    INFO] - Already cached /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-07 17:49:13,970] [    INFO] - Loading weights file model_state.pdparams from cache at /root/.paddlenlp/models/uie-senta-base/model_state.pdparams
[2023-09-07 17:49:14,628] [    INFO] - Loaded weights file from disk, setting weights to model.
W0907 17:49:14.640187  2030 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to install the corresponding wheel package according to the installation information on the official Paddle website.
W0907 17:49:14.640259  2030 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.0, Runtime API Version: 11.7
W0907 17:49:14.648037  2030 gpu_resources.cc:149] device: 0, cuDNN Version: 8.5.
LAUNCH INFO 2023-09-07 17:49:15,903 Exit code -6

![image](https://github.com/PaddlePaddle/Paddle/assets/29418975/ab1dd3d9-cb08-423d-8a22-053e9c6e4995)

两个命令都会报错

**Comment by will-jl944** [comment]
两个指令对应的的日志信息中都包含"The GPU architecture in your current machine is Pascal, which is not compatible with the Paddle installation with arch: 70 75 80 86"。你安装的Paddle不是为Pascal架构编译的。

可以参考[文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/install/compile/fromsource.html)编译匹配的paddle。

**Comment by zhangshenghang** [comment]
> 两个指令对应的的日志信息中都包含"The GPU architecture in your current machine is Pascal, which is not compatible with the Paddle installation with arch: 70 75 80 86"。你安装的Paddle不是为Pascal架构编译的。
> 
> 可以参考[文档](https://www.paddlepaddle.org.cn/documentation/docs/zh/install/compile/fromsource.html)编译匹配的paddle。

好的，更换了GPU架构，已经解决，感谢
